sublime_text.txt

```diff
@@ -0,0 +1,13 @@
+[Section]
+Name = Sublime Text 3
+Version = 3
+License = EULA
+Description = Code editor, powerful and lightweight.
+Category = 7
+URLSite = https://www.sublimetext.com/
+URLDownload = https://download.sublimetext.com/Sublime%20Text%20Build%203211%20Setup.exe
+SHA1 = a7f40c2bb09511d89c0ecead252a41f82b7e977b
+SizeBytes = 9840000
```

Comment:
This is still suspicously "even". Can you double-check the size to the last byte please?

Reply:
@JoachimHenze Sorry, I hadn't noticed that I didn't fix the byte size in sublime_text.txt